drift/lib/src/runtime/query_builder/expressions/null_check.dart

```diff
@@ -29,6 +29,21 @@ Expression<T> coalesce<T extends Object>(List<Expression<T>> expressions) {
   return FunctionCallExpression<T>('COALESCE', expressions);
 }
 
+/// Evaluates to the first expression that's not null, or null if both evaluate
+/// to null. See [coalesce] if you need more than 2.
+Expression<T> ifNull<T extends Object>(
+    Expression<T> first, Expression<T> second) {
+  return FunctionCallExpression<T>('IFNULL', [first, second]);
+}
+
+/// Extension defines the `nullIf` members.
+extension SqlNullIf<T extends Object> on Expression<T> {
+  /// Returns null if both [matcher] matches this expression.
+  Expression<T> nullIf(Expression<T> matcher) {
```

Comment:
Given that it's available on all expressions, let's actually make this a regular method on `Expression`.

Reply:
Done